Dockerfile

```diff
@@ -202,7 +202,7 @@ COPY --from=semgrep-core-container /src/semgrep/_build/default/src/main/Main.exe
 
 RUN ln -s semgrep-core /usr/local/bin/osemgrep
 
-# ???
+# Set some env variables
```

Comment:
you can remove your comment. I now those set some env variables, it's just it would be good to document where those env variable are used and for which purpose. For example it's inclear what  this SEMGREP_USER_AGENT_APPEND is doing. It seems redundant with SEMGREP_IN_DOCKER. If we know we are in docker, why we need yet another env variable that set to Docker?

Reply:
This is why I put those ??? so the person who know can add a comment explaining the purpose of those environment variables.